app/src/main/java/com/github/braillesystems/learnbraille/res/_DSL.kt

```diff
@@ -22,3 +22,8 @@ fun StepsBuilder.slateStylusLine(char: Char) {
         "Запишите на брайлевском приборе строку, состоящую из одного символа: $char."
     ).annotate(StepAnnotation.slateStylusRequired)
 }
+
+fun StepsBuilder.inputNumber(number: Int) {
+    +Input(content.markers.getValue(MarkerType.NumberSign))
```

Comment:
maybe annotate this step and make it possible to disable it in the settings?

Reply:
Viktoria Nikolaevna insists that we should teach to put number signs everywhere

Reply:
We have a special chat to validate VN proposals

Reply:
We shall discuss it tomorrow in the call

Reply:
decided not to do